packages/javascript-sdk/src/config/interfaces.ts

```diff
@@ -12,12 +12,25 @@ import type { ActionTypes } from './enums';
 import type { FRCallbackFactory } from '../fr-auth/callbacks/factory';
 import type { Tokens } from '../shared/interfaces';
 
+type LogLevel = 'none' | 'info' | 'error' | 'warn' | 'debug';
+
 interface Action {
   type: ActionTypes;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   payload: any;
 }
-
+/**
+ * Custom Logger for logger
+ */
+interface LoggerFunctions<
+  W = (...msgs: unknown[]) => void,
```

Comment:
defaulting this to a console.log/warn/error type, but this will allow a client to pass in their loggers type if needed, it may not be much different but i'm not sure how loggers type themselves and maybe this will provide more ergonomics if a client needs to use an FRLogger on their own?